front/lib/auth.ts

```diff
@@ -139,6 +143,41 @@ export class Authenticator {
   }
 }
 
+export async function personalWorkspace(
+  user: User
+): Promise<Result<Workspace, APIErrorWithStatusCode>> {
+  // Find the Workspace of type personal for which there is a Membership for `user`.
```

Comment:
I wonder if we should have a `creator` field on the workspace object itself, and not use any `Membership` for workspaces of type `"personal"`, so we could enforce at DB level that a user can only have one personal workspace.
Or is there some other way we could do it ?